task/buildah-min/0.6/buildah-min.yaml

```diff
@@ -581,6 +588,14 @@ spec:
         BUILDAH_ARGS+=("--inherit-labels=false")
       fi
 
+      if [ -n "${BUILDAH_SOURCE_DATE_EPOCH}" ]; then
+        BUILDAH_ARGS+=("--source-date-epoch=${BUILDAH_SOURCE_DATE_EPOCH}")
+        BUILDAH_ARGS+=("--rewrite-timestamp")
```

Comment:
Note that this is a separate option for a reason; it's a big hammer, and very crucially doesn't cover things like Python files with embedded timestamps - which are one of the things currently breaking rhel-bootc reproducibility. To solve things like that we need to wire up https://github.com/keszybz/add-determinism or similar.

(really what we want is a combination of add-determinism and layer splitting)